packages/repo-collector-types/src/index.ts

```diff
@@ -125,6 +137,10 @@ export interface WebappMetricDataRepoDatapoint {
       browseUrl: string
     }[]
   }
+  sonarCloud?: {
+    enabled: boolean // false means that no project has been set up for the GitHub repository
```

Comment:
tip: if you want to have comments here, it can be useful to use tsdoc instead so that we'll get this information in our IDEs when referencing the types
```
sonarCloud?: {
  /**
   * Describes whether SonarCloud is configured for the GitHub repository in question.
   */
  enabled: boolean
}
```